### PR TITLE
Add public getModelDownloadId method 

### PR DESCRIPTION
**Pull request by annzimmer**
Can be used to track download progress.

### CODE REVIEW

File: firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/CustomModelDownloadConditions.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public boolean isDeviceIdleRequired() {
     private boolean isWifiRequired = false;
     private boolean isDeviceIdleRequired = false;
 
-    /** Sets whether charging is required. Only works on Android N and above. */
+    /** Sets charging is required. Only works on Android N and above. */
```
Comment (rlazo) [inline firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/CustomModelDownloadConditions.java:57]:
```suggestion
    /** Sets charging as required. Only works on Android N and above. */
```

File: firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/CustomModelDownloadConditions.java [hunk 2]
```diff
@@ -63,15 +63,15 @@ public Builder requireCharging() {
       return this;
     }
 
-    /** Sets whether wifi is required. */
+    /** Sets wifi is required. */
```
Comment (rlazo) [inline firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/CustomModelDownloadConditions.java:66]:
```suggestion
    /** Sets wifi as required. */
```

File: firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/CustomModelDownloadConditions.java [hunk 3]
```diff
@@ -63,15 +63,15 @@ public Builder requireCharging() {
...
     @NonNull
     public Builder requireWifi() {
       this.isWifiRequired = true;
       return this;
     }
 
     /**
-     * Sets whether device idle is required.
+     * Sets device idle is required.
```
Comment (rlazo) [inline firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/CustomModelDownloadConditions.java:74]:
```suggestion
     * Sets device idle as required.
```

**Comment by annzimmer** [comment]
PTAL: Applied reviewer suggested changes.

**Comment by rlazo** [comment]
The issue with the copyright-check have been fixed in master. Please, merge it to this branch to ensure all tests pass. Thanks!

**Review by rlazo — APPROVED** [review]

**Pull request merged**